omspy/brokers/api_helper.py

```diff
@@ -1,56 +1,56 @@
-from NorenRestApiPy.NorenApi import  NorenApi
+from NorenRestApiPy.NorenApi import NorenApi
 from threading import Timer
 import pandas as pd
 import time
 import concurrent.futures
 
 api = None
-class Order:
```

Comment:
This is legacy code from an another provider.
Except for formatting, we won't be changing anything here